bindings/ruby/fdb.gemspec.in

```diff
@@ -15,8 +15,8 @@ EOF
   s.email = 'fdb-dist@apple.com'
   s.files = ["LICENSE", "lib/fdb.rb", "lib/fdbdirectory.rb", "lib/fdbimpl.rb", "lib/fdblocality.rb", "lib/fdboptions.rb", "lib/fdbsubspace.rb", "lib/fdbtuple.rb", "lib/fdbimpl_v609.rb"]
   s.homepage = 'https://www.foundationdb.org'
-  s.license = 'Apache v2'
-  s.add_dependency('ffi', '>= 1.1.5')
+  s.license = 'Apache-2.0'
+  s.add_dependency('ffi', '~> 1.1', '>= 1.1.5')
```

Comment:
This version dates back to 2012, and I think there's some desire to update the supported Ruby versions to be more modern ones as well (see https://github.com/apple/foundationdb/issues/3259).

Since we're making this change to master and it won't affect any bindings in the near future, I think it would be reasonable to do either 1) adopt the above change to silence the warning and make a note in the other PR to pick a more sensible version or 2) defer this change to the other PR.

Reply:
@vishesh, can you take a look?  I don't know anything about Ruby, but rubygems told me to make the above changes if I wanted to silence some build warnings.

Reply:
I don't think there's anything more to look at here, I think we should either just go ahead and merge while adding a note to deferring the changes to the other issue, or just wait for the other issue to be tackled. Plus, I think you already merged these changes in another PR: #3342.

Reply:
I added a note on the other issue, I think we’re good now to keep the changes already merged in #3342 and just close this one. Feel free to reopen if I’ve made a mistake.